docs/samples/s3/s3_secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Should we provide yaml for the service account?

Reply:
yes we can do that